plugins/woocommerce/includes/wc-rest-functions.php

```diff
@@ -139,7 +139,7 @@ function wc_rest_set_uploaded_image_as_attachment( $upload, $id = 0 ) {
 		include_once ABSPATH . 'wp-admin/includes/image.php';
 	}
 
-	$image_meta = wp_read_image_metadata( $upload['file'] );
+	$image_meta = @wp_read_image_metadata( $upload['file'] );
```

Comment:
Adding the `@` here will suppress any warnings from calling this function. I noticed other calls to `wp_read_image_metadata` within WooCommerce are already doing this.